src/main/java/jenkins/plugins/office365connector/Webhook.java

```diff
@@ -34,7 +38,9 @@
     public static final Integer DEFAULT_TIMEOUT = 30000;
 
     private String name;
+    // The url can be specified using either a string or the id of plain text credentials
     private String url;
+    private String urlCredentialsId;
```

Comment:
Did you look into references documentation to remove second filed for url and to have only the old one?